lib/cisco_node_utils/yum.rb

```diff
@@ -20,8 +20,9 @@
 require_relative 'node_util'
 
 module Cisco
-  # This Yum class provides cisco package management functions through nxapi.
+  # This Yum class provides cisco package management functions.
   class Yum < NodeUtil
+    EXEC_IN_DEFAULT_NS = 'ip netns exec default sdr_instcmd'
     def self.decompose_name(file_name)
```

Comment:
Can you add a comment describing what this function is expected to return? Looks like the match-group (package_name, package_version, os_version, ???)